mbed_lstools/lstools_base.py

```diff
@@ -206,12 +209,13 @@ def _update_device_from_htm(self, device):
             device.update({"daplink_%s" % f.lower().replace(' ', '_'): v
                            for f, v in daplink_info.items()})
         if htm_target_id:
-            logging.debug("Found htm target id, %s, for usb target id %s",
+            logger.debug("Found htm target id, %s, for usb target id %s",
                             htm_target_id, device['target_id_usb_id'])
             device['target_id'] = htm_target_id
         else:
-            logging.warning("Could not read htm on from usb id %s. "
-                            "Falling back to usb id",
+            logger.warning("No target ID found inside of the redirect URL in"
+                            " file %s. Falling back to usb id %s",
+                            join(mount_point, self.MBED_HTM_NAME),
```

Comment:
Jimmeh, `mount_point` doesn't exist here. I think you meant to index into `device`.

But the fact the test passed means that we don't have any coverage for this case either.

Reply:
lol. no coverage!

Reply:
I should really test these things.

Reply:
I'll have to get back to this tomorrow. I have a website to deploy....

Reply:
No worries, I realize I'm adding more to your TODO list 😛 Thanks for keeping at it!